README.md

```diff
@@ -33,7 +33,8 @@ error: Delete `;` (prettier/prettier) at pkg/commons-atom/ActiveEditorRegistry.j
 ## Installation
 
 ```sh
-npm install --save-dev prettier eslint-plugin-prettier
+npm install --save-dev eslint-plugin-prettier
+npm install --save-dev --save-exact prettier
```

Comment:
Exact installation is recommended for `prettier` https://prettier.io/docs/en/install.html